CONTRIBUTING.md

```
@@ -1 +1,135 @@
+	```bash
+	# root directory
+	bunx @changesets/cli
+	```
+	Follow the wizard and commit the changes.
+
+6. **Create a pull request**:
+
+
+## Pull Request Review Process
+
+1. **Ask coderabbit for a review**:
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)